clients/v1_repl/v1_repl.go

```diff
@@ -0,0 +1,912 @@
+			historyFile.WriteString(cmd + "\n")
+			historyFile.Close()
+		}
+	}
+}
+
+func (c *Client) Create(ctx context.Context) error {
+	res, err := c.GetPing(ctx).ExecuteWithHttpInfo()
+	if err != nil {
+		color.Red("Unable to connect to InfluxDB")
+		return err
+	}
+	build := res.Header.Get("X-Influxdb-Build")
+	version := res.Header.Get("X-Influxdb-Version")
+	color.Cyan("Connected to InfluxDB %s %s", build, version)
+
+	// compute historyFilePath at REPL start
+	// Only load/write history if HOME environment variable is set.
+	var historyDir string
+	if runtime.GOOS == "windows" {
+		if userDir := os.Getenv("USERPROFILE"); userDir != "" {
+			historyDir = userDir
+		}
+	}
+	if homeDir := os.Getenv("HOME"); homeDir != "" {
+		historyDir = homeDir
+	}
+	if historyDir != "" {
+		c.historyFilePath = filepath.Join(historyDir, ".influx_history")
+	}
+
+	p := prompt.New(c.executor,
+		c.completer,
+		prompt.OptionTitle("InfluxQL Shell"),
+		prompt.OptionHistory(c.readHistory()),
+		prompt.OptionDescriptionTextColor(prompt.Cyan),
+		prompt.OptionPrefixTextColor(prompt.Green),
+		prompt.OptionCompletionWordSeparator(" ", "."),
+	)
+	c.Databases, _ = c.GetDatabases(ctx)
+	p.Run()
+	return nil
+}
+
+var allInfluxQLKeywords []prompt.Suggest = []prompt.Suggest{
+	// * Commented out are unsupported keywords in 2.x
+	{Text: "ALL"},
+	// {Text: "ALTER"},
+	{Text: "ANY"},
+	{Text: "AS"},
+	{Text: "ASC"},
+	{Text: "BEGIN"},
+	{Text: "BY"},
+	// {Text: "CREATE"},
+	{Text: "CONTINUOUS"},
+	{Text: "DATABASE"},
+	{Text: "DATABASES"},
+	{Text: "DEFAULT"},
+	{Text: "DELETE"},
+	{Text: "DESC"},
+	{Text: "DESTINATIONS"},
+	{Text: "DIAGNOSTICS"},
+	{Text: "DISTINCT"},
+	{Text: "DROP"},
+	{Text: "DURATION"},
+	{Text: "END"},
+	{Text: "EVERY"},
+	{Text: "EXPLAIN"},
+	{Text: "FIELD"},
+	{Text: "FOR"},
+	{Text: "FROM"},
+	// {Text: "GRANT"},
+	{Text: "GRANTS"},
+	{Text: "GROUP"},
+	{Text: "GROUPS"},
+	{Text: "IN"},
+	{Text: "INF"},
+	{Text: "INSERT", Description: "Insert line protocol data"},
+	{Text: "INTO"},
+	{Text: "KEY"},
+	{Text: "KEYS"},
+	// {Text: "KILL"},
+	{Text: "LIMIT"},
+	{Text: "SHOW"},
+	{Text: "MEASUREMENT"},
+	{Text: "MEASUREMENTS"},
+	{Text: "NAME"},
+	{Text: "OFFSET"},
+	{Text: "ON"},
+	{Text: "ORDER"},
+	{Text: "PASSWORD"},
+	{Text: "POLICY"},
+	{Text: "POLICIES"},
+	{Text: "PRIVILEGES"},
+	{Text: "QUERIES"},
+	{Text: "QUERY"},
+	{Text: "READ"},
+	{Text: "REPLICATION"},
+	{Text: "RESAMPLE"},
+	{Text: "RETENTION"},
+	// {Text: "REVOKE"},
+	{Text: "SELECT"},
+	{Text: "SERIES"},
+	{Text: "SET"},
+	{Text: "SHARD"},
+	{Text: "SHARDS"},
+	{Text: "SLIMIT"},
+	{Text: "SOFFSET"},
+	{Text: "STATS"},
+	{Text: "SUBSCRIPTION"},
+	{Text: "SUBSCRIPTIONS"},
+	{Text: "TAG"},
+	{Text: "TO"},
+	{Text: "USER"},
+	{Text: "USERS"},
+	{Text: "VALUES"},
+	{Text: "WHERE"},
+	{Text: "WITH"},
+	{Text: "WRITE"},
+}
+
+var replKeywords []prompt.Suggest = []prompt.Suggest{
+	// TODO: provide a way to connect to a new endpoint during the same REPL session
```

Comment:
Is this covered in #389? If not we should make an issue for it as well

Reply:
Same issue :)